modules/veteran/app/controllers/veteran/v0/representatives_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Veteran
+  module V0
+    class RepresentativesController < ApplicationController
+      skip_before_action :set_tags_and_extra_content, raise: false
+      skip_before_action :authenticate
+
+      def search
+        rep = Veteran::Service::Representative.for_user(target)
+        if rep.present?
+          render json: { data: rep.as_json }
+        else
+          render json: { errors: [{ detail: 'Representative not found' }] }
+        end
+      end
+
+      private
+
+      def target
```

Comment:
I wonder if we shouldn't just change the `for_user` to take first_name, last_name. Seems silly to have to build this to make it work

Reply:
The for_user method takes in a DOB and a SSN possibility as well. We could make them all KWA but if we want to add more possibilities down the road like organization name, then the arg list will get long. Thoughts?

Reply:
The method has been changed to KWA